A3A/addons/core/functions/init/fn_initVehClassToCrew.sqf

```diff
@@ -34,21 +34,21 @@ FIX_LINE_NUMBERS()
 private _allVehClassToCrew = [
 // Vehicles categories at the top have higher priority than bellow.
 // So if "Tank_F" is in both NATOLand and NATOTanks, NATOTanks should be ABOVE NATOLand, as NATOTanks is a specialised child.
-    [FactionGet(all,"vehiclesRivalsArmor"),[FactionGet(occ,"unitCrew"), FactionGet(riv,"unitCrew"), FactionGet(reb,"unitCrew"), "C_Man_1"]],
-    [FactionGet(all,"vehiclesRivalsLight"),[FactionGet(occ,"unitRifle"), FactionGet(riv,"unitRifle"), FactionGet(reb,"unitRifle"), "C_Man_1"]],
-    [FactionGet(all,"vehiclesRivalsStatics"),[FactionGet(occ,"unitRifle"), FactionGet(riv,"unitRifle"), FactionGet(reb,"unitRifle"), "C_Man_1"]],
-    [FactionGet(all,"vehiclesRivals"),[FactionGet(occ,"unitRifle"), FactionGet(riv,"unitRifle"), FactionGet(reb,"unitRifle"), "C_Man_1"]],
-    [FactionGet(all,"vehiclesRivalsHelis"),[FactionGet(occ,"unitRifle"), FactionGet(riv,"unitRifle"), FactionGet(reb,"unitRifle"), "C_Man_1"]],    
+    [FactionGet(all,"vehiclesMilitia"), [FactionGet(occ,"unitMilitiaGrunt"), /* FactionGet(inv,"unitMilitiaGrunt"), */ FactionGet(reb,"unitCrew"), FactionGet(civ,"unitMan")]],
+    [FactionGet(all,"vehiclesRivalsArmor"),[FactionGet(occ,"unitCrew"), FactionGet(inv,"unitCrew"), FactionGet(riv,"unitCrew"), FactionGet(reb,"unitCrew"), FactionGet(civ,"unitMan")]],
+    [FactionGet(all,"vehiclesRivalsLight"),[FactionGet(occ,"unitRifle"), FactionGet(inv,"unitRifle"), FactionGet(riv,"unitRifle"), FactionGet(reb,"unitRifle"), FactionGet(civ,"unitMan")]],
+    [FactionGet(all,"vehiclesRivalsStatics"),[FactionGet(occ,"unitRifle"), FactionGet(inv,"unitRifle"), FactionGet(riv,"unitRifle"), FactionGet(reb,"unitRifle"), FactionGet(civ,"unitMan")]],
+    [FactionGet(all,"vehiclesRivals"),[FactionGet(occ,"unitRifle"), FactionGet(inv,"unitRifle"), FactionGet(riv,"unitRifle"), FactionGet(reb,"unitRifle"), FactionGet(civ,"unitMan")]],
+    [FactionGet(all,"vehiclesRivalsHelis"),[FactionGet(occ,"unitRifle"), FactionGet(inv,"unitRifle"), FactionGet(riv,"unitRifle"), FactionGet(reb,"unitRifle"), FactionGet(civ,"unitMan")]],    ///why FactionGet(all and not riv? why c_man and not  FactionGet(civ,"unitMan")? why no FactionGet(inv ?
```

Comment:
`FactionGet(all` is probably something to do with a hashmap, don't change it
Don't add Inv if you don't need to, it'll more than likely have unintended consequences (again, don't change it)

As for the `(civ,"unitMan")`, nobody expected to need a civ that wasn't vanilla probably. Should be fine

Reply:
I'll experiment a bit more, but in general I figured out that if vehicles present both in main faction and in rivals, rivals crew would spawn even if vehicle is spawned during invaders QRFLAND, though I fixed it by moving Militia vehicles to the top, maybe I'll try moving whole rivals section to the bottom and see what happens.